public/vendor/fontawesome/less/variables.less

```diff
@@ -1,17 +1,21 @@
 // Variables
 // --------------------------
 
-@fa-font-path:        "./vendor/fontawesome/fonts";
+@fa-font-path:        "../fonts";
```

Comment:
I believe this line will need to be reverted, otherwise fonts will not show up on client-side.